tools/bot/pkg/util/grpcutil/grpc.go

```diff
@@ -0,0 +1,78 @@
+package grpcutil
```

Comment:
please use https://github.com/iotexproject/iotex-antenna-go
it already has these codes